test/contrib/test_module.py

```diff
@@ -0,0 +1,43 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
+import haiku as hk
+import flax
+from numpyro.contrib.module import haiku_module, flax_module
+import numpy as np
+from numpyro import handlers
+import numpyro
+
+
+X = np.array(list(range(100)))
+Y = np.array([x*2 + 2.0 for x in list(range(100))])
```

Comment:
nit: `X = np.arange(100)`, `Y = 2 * x + 2`, no need for `param_keys`. I think you can move those global variables to each of `test_flax_module`, `test_haiku_module`.